CODE_OF_CONDUCT.md

```diff
@@ -14,24 +14,24 @@ We document everything that is relevant for other developers
 
 In a PR we avoid to change established conventions. We will challenge it in a separate meeting.
 
-A PR should be merged as soon as possible and not be a place for comprehensive conversation. If a conversation gets too intensive finish it bilaterally and offline. After agreement is done, we write it down and reference it in the PR.
+**A PR should be merged as soon as possible and not be a place for comprehensive conversation**. If a conversation gets too intensive finish it bilaterally and offline. After agreement is done, we write it down and reference it in the PR.
 
-We always respects each others opinion as every opinion is valid. The better idea always wins, even if it comes from an apprenticeship or from a CTO. If a clash on opinion happens, we discuss it in a constructive way.
+**We always respects each others opinion as every opinion is valid**. The better idea always wins, even if it comes from an apprenticeship or from a CTO. If a clash on opinion happens, we discuss it in a constructive way.
 
 We assume that a PR creator knows Standard HTML, CSS and JS. We wont explain basics in a PR but are very happy to help verbally and bilaterally those new future frontend heroes. Everyone of us started from the beginning and helping someone is a honor for us.
 
 An example of constructive conversation can look like:
 
 * DONT: This is wrong, here the link to specs
-* DO: I'd like to refer to our coding style defined in our XXX, here, this xxxx would be correct. OR because of...
+* DO: I'd like to refer to our coding style defined in our XXX, here, this xxxx would be correct because of...
 
-We aim to be as efficient as possible. On a PR review, we try to explain before referring to specs as we assume everyone have read the specs but maybe not understood it. We use link to specs as a nice add-on, not as a minimum argument. Of course, if explanation requires lots of time, just referring to specs might be ok. As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible. We also assume that everyone is willing to learn. Teaching is very welcome but is based on good will.
+**We aim to be as efficient as possible.** On a PR review, we try to explain before referring to specs as we assume everyone have read the specs but maybe not understood it. We use link to specs as a nice add-on, not as a minimum argument. Of course, if explanation requires lots of time, just referring to specs might be ok. As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible. We also assume that everyone is willing to learn. Teaching is very welcome but is based on good will.
```

Comment:
I think it's not only about refering to specs, but rather documentations, tutorials, articles, etc..
There other great resources online, like MDN, Medium, CSSTricks, CSSWizardy, Google Devs, etc.
I appreciate these a lot and want to use and reference them.

Reply:
the idea is use links always as an add-on not as the only one (of course where it make sense and if u have time)

Reply:
But how about code examples? or reproduction repos?
These are not covered but very helpfull.

Reply:
gists, codesandbox, etc

Reply:
I think the sentence: "As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible." covers it and also includes what u said. Or not?

Reply:
Okay